extensions/wikia/AdEngine/i18n/AdEngine.i18n.php

```diff
@@ -0,0 +1,12 @@
+<?php
+$messages = [];
+
+$messages[ 'en' ] = [
+	'adengine-recovery-message-blocking-welcome' => 'Hey! It looks like you\'re using ad blocking software!',
+	'adengine-recovery-message-blocking-message' => 'Wikia runs ads so we can keep the lights on and so Wikia will always be free to use. We can bring you fun, free, fan-oriented content until my glorious return to Earh if you <strong>add us to your adblock whitelist</strong>. <a class="action-accept">Click</a> my face to refresh after you\'re done!',
```

Comment:
@Grunny @lgarczewski @adamkarminski do you have any idea how to get rid-off the HTML here? I recall it violates i18n good practices. I think we could use WikiText to bold the text within `<strong />` tag but I don't know what to do with the anchor...